ethereumj-core/src/main/java/org/ethereum/net/rlpx/HandshakeHandler.java

```diff
@@ -252,6 +256,8 @@ private void decodeHandshake(final ChannelHandlerContext ctx, ByteBuf buffer) th
 
                 final HelloMessage inboundHelloMessage = (HelloMessage) message;
                 channel.setNode(remoteId, inboundHelloMessage.getListenPort());
+                // use node into discovery
+                nodeManager.getNodeHandler(channel.getNode());
```

Comment:
I suppose the line above `setNode()` should do the same (i.e. `getNodeHandler` should be invoked)
Can we remove this then?